src/input.rs

```diff
@@ -61,6 +61,34 @@ impl<Manager> Input<Manager> {
         unsafe { sys::SteamAPI_ISteamInput_GetActionSetHandle(self.input, name.as_ptr()) }
     }
 
+    /// Returns the input type for a controler
+    pub fn get_input_type_for_handle(
+        &self,
+        input_handle: sys::InputHandle_t,
```

Comment:
Not happy that we're directly exposing `steamworks-sys` types, but this can be handled in another PR.

Reply:
How's that?

Reply:
sorry, to be clear: I've added a new enum to wrap the `steamworks-sys` type, let me know if that's ok :)